google-cloud-firestore/src/main/java/com/google/cloud/firestore/FieldPath.java

```diff
@@ -155,6 +155,6 @@ public String toString() {
 
   @Override
   public int compareTo(@Nonnull FieldPath other) {
-    return getEncodedPath().compareTo(other.getEncodedPath());
+    return super.compareTo(other);
```

Comment:
We can technically just not override this altogether right? If you want the delegation to superclass to be explicit, I'm OK with it too.

Reply:
I was debating between the two options, but I am fine with removing it.